### PR TITLE
Adding new section about obs external query

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/13358

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -262,6 +263,43 @@ See xref:../observability/customize_observability.adoc#forward-alerts[Forwarding
...
+Observability provides an external API for metrics to be queried through the OpenShift route, `rbac-query-proxy`. View the following tasks to use `rbac-query-proxy` route:
+
+* You can get the details of the route with the following command:
++
+----
+oc get route rbac-query-proxy -n open-cluster-management-observability
+----
+
+* To access the `rbac-query-proxy` route, you must have an OpenShift OAuth access token. The token should be associated with a user or service account, which has permission to get namespaces. For more information, see https://docs.openshift.com/container-platform/4.8/authentication/managing-oauth-access-tokens.html[Managing user-owned OAuth access tokens].
+//Get the default CA certificate to store the content of the key, `tls.crt`, OR Get the default CA certificate and store the content of the key, `tls.crt`,
```
Comment (dockerymick) [inline observability/observability_enable.adoc]: @marcolan018 I left a clarification question on line 278

Comment (marcolan018) [reply observability/observability_enable.adoc]: Get the default CA certificate and store the content of the key, `tls.crt`

**Comment by dockerymick** [comment]
@marcolan018 Thanks for the review and clarification! Going to merge the PR

**Pull request merged**